src/OidcClient.php

```diff
@@ -94,20 +94,21 @@ public function authenticate(Request $request): OidcTokens
 
     // Request and verify the tokens
     return $this->verifyTokens(
-        $this->requestTokens('authorization_code', $code, $this->getRedirectUrl())
+        $this->requestTokens('authorization_code', $code, $this->getRedirectUrl()),
+        $verifyNonce
     );
   }
 
   /** {@inheritDoc} */
-  public function refreshTokens(string $refreshToken): OidcTokens
+  public function refreshTokens(string $refreshToken, bool $verifyNonce = false): OidcTokens
```

Comment:
This change can be reverted, nonce is not supported at all for the refresh action.

Reply:
Done.